src/main/java/net/codedstingray/worldshaper/block/variable/VariableMap.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * WorldShaper, a powerful in-game map editing and terraforming tool for Minecraft.
+ * Copyright (C) 2024 CreativeConduit
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.codedstingray.worldshaper.block.variable;
+
+import net.codedstingray.worldshaper.block.BlockState;
+import net.codedstingray.worldshaper.block.mask.Mask;
+import net.codedstingray.worldshaper.util.vector.vector3.Vector3i;
+import org.bukkit.Material;
+import org.bukkit.World;
+import org.bukkit.block.Block;
+import org.bukkit.block.data.BlockData;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static net.codedstingray.worldshaper.util.world.LocationUtils.vectorToLocation;
+
+public class VariableMap {
```

Comment:
missing docs

Reply:
Done. Also fixed a method name in the process